Client/Frontend/Browser/Tab.swift

```diff
@@ -63,6 +63,22 @@ class Tab: NSObject {
     var pendingScreenshot = false
     var url: URL?
 
+    fileprivate var _noImageMode = false
+
+    @available(iOS 11, *)
+    var noImageMode: Bool {
```

Comment:
AFAIK, we don't follow this pattern anywhere else. It might be more "swifty" to do something like:

```
var noImageMode: Bool {
  didSet {
    if noImageMode == oldValue {
      return
    }

    let helper = (contentBlocker as! ContentBlockerHelper)
    helper.noImageMode(enabled: noImageMode)
  }
}
```

Reply:
Agreed, but this is a quirk, `didSet` won't combine with `available`. `didSet` doesn't change the type of the property to computed, and only computed props -being functions- can be wrapped in `available`.

Reply:
Ugh. That sucks. In that case, what you have should be fine. Maybe leave a comment above explaining why this pattern needed to be used.